ImportBlocks/ImportBlocks.cs

```diff
@@ -17,23 +17,14 @@ public override void Configure()
             Settings.Load(GetConfiguration());
         }
 
-        private bool OnExport(string[] args)
```

Comment:
maybe keep the `OnExport` function declaration before `WriteBlocks`.

Reply:
will do